### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by rickycraft**
I thought docker could be useful.

### CODE REVIEW

File: package.json
Comment (eutampieri) [inline package.json:8]: Please don't use sudo since `usermod -aG docker` is a much nicer way

File: docker-compose.yml
Comment (eutampieri) [inline docker-compose.yml:12]: Good, I assume this is to ease the configuration since you already configured the right volumes.

File: index.js
Comment (eutampieri) [inline index.js:59]: Actually, an environment variable with localhost as a fallback would be better than defaulting to ::/0

File: package.json
Comment (eutampieri) [inline package.json:32]: Add newline at the end of file

File: index.js
Comment (eutampieri) [inline index.js:61]: We like newlines at the end of files

File: .dockerignore
Comment (eutampieri) [inline .dockerignore:3]: Newline

**Review by eutampieri — CHANGES_REQUESTED** [review]
Good job, maybe resolve the conversations. Waiting for @FrancescoBonzi 's feedback too.

File: index.js
Comment (rickycraft) [inline index.js:31]: If you have a suggestion for a better variable name, i'm up to it

File: docker-compose.yml
Comment (rickycraft) [reply docker-compose.yml:12]: Maybe adding a mention on the README could be useful. Also maybe providing also a plain docker command.

File: Dockerfile
Comment (rickycraft) [inline Dockerfile:1]: Alpine image is lighter and works the same from my testing

Comment (rickycraft) [reply Dockerfile:1]: node 18 is currently LTS and same as before it works fine

**Comment by eutampieri** [comment]
ok
Il giorno 05/nov/22, alle ore 15:42, Riccardo Roveri ha scritto:

**Comment by eutampieri** [comment]
I wholly agree and approove. Alpine FTW!

**Comment by eutampieri** [comment]
Maybe BIND_ADDR?
Il giorno 05/nov/22, alle ore 15:38, Riccardo Roveri ha scritto:

File: index.js
Comment (eutampieri) [reply index.js:31]: BIND_ADDR

**Review by eutampieri — CHANGES_REQUESTED** [review]
Almost there ;)

**Comment by rickycraft** [comment]
Do you want to update the README with docker infos?

**Comment by eutampieri** [comment]
> Do you want to update the README with docker infos?

This is up to @FrancescoBonzi

**Review by eutampieri — APPROVED** [review]

File: .dockerignore
Comment (eutampieri) [inline .dockerignore:2]: Add .git

**Review by eutampieri — CHANGES_REQUESTED** [review]

**Comment by FrancescoBonzi** [comment]
Grazie mille @rickycraft per il lavoro!! Super apprezzato 😊

**Pull request merged**
